lib/logger.dart

```diff
@@ -30,8 +30,10 @@ class BreezLogger {
     if (kDebugMode) {
       Logger.root.onRecord.listen((record) {
         // Dart analyzer doesn't understand that here we are in debug mode so we have to use kDebugMode again
-        if (kDebugMode) {
-          print(_recordToString(record));
+        if (record.level >= Level.INFO) {
```

Comment:
I think the sdk signer and node logs have debug level. We do need to include them by default.

Reply:
This filter is only applied for printing (the log file will keep all logs); Do you want to print these to log too @roeierez ?

Reply:
Is there a reason to differentiate?

Reply:
My rational is the bug report as I mention here: https://github.com/breez/c-breez/issues/681#issuecomment-1766553792

I'll remove from the file; in future we can add back if we want.

@erdemyerebasmaz I'll keep the naming changes to do not lose it; different from the sdk logs, the app logs are very few and add a lot of information about user behavior, essential on bug reports